src/strategies/ticket-validity/index.ts

```diff
@@ -0,0 +1,25 @@
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'ethedev';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const score = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+  return Object.fromEntries(
+    Object.entries(score).map((address: any) => [address[0], (address[1] > 0 ? 1 : 0)])
```

Comment:
It's better if the minimum is a parameter like this
```suggestion
    Object.entries(score).map((address: any) => [address[0], (address[1] > options.min || 0 ? 1 : 0)])
```